mmv1/third_party/terraform/tests/resource_compute_network_endpoint_group_test.go

```diff
@@ -31,6 +31,33 @@ func TestAccComputeNetworkEndpointGroup_networkEndpointGroup(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
```

Comment:
This file needs to be renamed to end with `.go.erb` now that it has embedded ruby in it.